literature.bib

```diff
@@ -14301,6 +14416,14 @@ @inproceedings{YGM:SEAS06
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2005 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@book{MS05,
+  title = {{Software Ecosystem: Understanding an Indispensable Technology and Industry}},
+  author = {{Messerschmitt, David G and Szyperski, Clemens}},
+  volume = {1},
+  year = 2005,
+  publisher = MITpress
```

Comment:
address missing

Reply:
There are two addresses in the book direcltly next to each other, `CambridgeMIT` and `London`. What is out policy in such a case?

Reply:
I just pick `CambridgeMIT` now as both are fine.